src/app.ts

```diff
@@ -88,6 +88,7 @@ import { temporarilyBlockedRouter } from "./components/account-intervention/temp
 import { resetPassword2FAAuthAppRouter } from "./components/reset-password-2fa-auth-app/reset-password-2fa-auth-app-routes";
 import { setGTM } from "./middleware/analytics-middleware";
 import { setCurrentUrlMiddleware } from "./middleware/current-url-middleware";
+import { setApiRateLimiter } from "./middleware/api-rate-limiter-middleware";
```

Comment:
@whi-tw will this see the user's ip address or the cloudfront ip address?

Reply:
Please don't merge until we've reviewed the rate limiting.

Reply:
sorry, didn't see the comment.

as-per slack:

> WAF makes more sense - no point making express do it
> it should see the right IP, as it will likely use x-forwarded-for, but it's best not to use anything IP'y if we can help it.